app/backend/schema/droneSchema.ts

```diff
@@ -0,0 +1,23 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+enum DroneStatus {
+    "ON_WAY_TO_DELIVERY",
+    "ON_WAY_BACK_FROM_DELIVERY",
+    "IDLE",
+    "CHARGING",
+    "MAINTENACE"
+}
+
+const droneSchema = new Schema({
```

Comment:
Do we want a location for the drone?

Reply:
when we do location its probs gonna be latitude/longitude i assume. i also thought that the drone API would take care of most it...?